Xwt/Xwt/TreeStore.cs

```diff
@@ -69,6 +69,11 @@ public TreeStore (params IDataField[] fields)
 		ITreeStoreBackend Backend {
 			get { return (ITreeStoreBackend)BackendHost.Backend; }
 		}
+
+		public void SetSortField (IDataField field, ColumnSortDirection order)
```

Comment:
Well, I don't have any immediate interest in getting this merged (anymore), so since it seems to have bitrotted, I'd say whoever feels the need should feel free to pick this work up and finish it.